tensorflow/core/grappler/optimizers/auto_mixed_precision.cc

```diff
@@ -56,6 +59,22 @@ const char kCastToFp16[] = "CastToFp16";
 const char kCastToBf16[] = "CastToBf16";
 const char kCastToFp32[] = "CastToFp32";
 
+#if TENSORFLOW_USE_ROCM
+std::vector<std::string> FP16SupportedDevices = 
+{
+    "906", 
+    "908", 
+    "90a"
```

Comment:
@mvermeulen , can we put "90a" in open source yet?

Reply:
I have a number of merge commits with develop-upstream. Rebasing and squashing is a bit difficult with all of these. @deven-amd